src/plugin/PluginManager.php

```diff
@@ -434,7 +434,17 @@ public function enablePlugin(Plugin $plugin) : void{
 			$this->server->getLogger()->info($this->server->getLanguage()->translate(KnownTranslationFactory::pocketmine_plugin_enable($plugin->getDescription()->getFullName())));
 
 			$plugin->getScheduler()->setEnabled(true);
-			$plugin->onEnableStateChange(true);
+			try{
+				$plugin->onEnableStateChange(true);
+			}catch(DisablePluginException $e){
+				$errorMessage = $e->getMessage();
+				if($errorMessage !== ""){
+					$plugin->getLogger()->error($errorMessage);
+				}
```

Comment:
I think it would be better to omit this, and let the plugin itself do any error logging, so that it can localize any messages as needed. Passing a message to DisablePluginException doesn't make much sense since you can't translate.

Reply:
What prevents the plugin from providing a translatable message?

Reply:
They aren't able to inject language strings into any language context that the server knows about. It's better to just let them log the way they normally would.

Reply:
I said *translated* not *translatable*

Reply:
My bad, did not see that.